src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -21,7 +23,7 @@ module.exports = function(pixels, highThresholdRatio, lowThresholdRatio, useHyst
     grads.push([]);
     angles.push([]);
     for (var y = 0; y < pixels.shape[1]; y++) {
-      var result = sobelFilter(
+      var result = sobelFilter( // Convolves the sobel filter on every pixel.
```

Comment:
Please remove full stop from phrases

Reply:
Why is that considered as a phrase?

Reply:
I supposed that a sentence consists of a subject and predicate. Here only predicate is present, hence I thought that it is a phrase.

Reply:
```suggestion
      var result = sobelFilter( // Convolves the sobel filter on every pixel
```